common/ellipsis.js

```diff
@@ -217,64 +213,83 @@
-                        if (containsOverflow == false) {
-
-                            //Iterate over table data in the row
-                            for (var i = 0; i < element[0].children.length; i++) {
-
-                                // Get all images and iframes inside the td
-                                var imagesAndIframes = UiUtils.getImageAndIframes(element[0].children[i]);
-
-                                // Bind onload event and updateheight for particular td index
-                                imagesAndIframes.forEach(function(el) {
-                                    var index = i;
-                                    el.onload = function() {
-                                        updateHeight(index, el);
-                                    };
-                                });
+                    var initializeOverflowLogic = function () {
+                        setOverflows();
+
+                        scope.hideContent = true;
+                        scope.maxHeightStyle = maxHeightStyle;
+
+                        scope.$digest();
+                    }
 
-                                updateHeight(i, element[0].children[i].children[0]);
+                    new ResizeSensor(element[0], function (dimensions) {
+                        // if TR.offsetHeight > the calculated maxRecordsetRowHeight
+                        // +10 to account for padding on TD element
+                        if (dimensions.height > (maxHeight + 10)) {
+                            // iterate over each cell (TD), check it's height, and set overflow if necessary
+                            if (!userClicked) {
+                                initializeOverflowLogic();
                             }
+                        } else if (dimensions.height < (maxHeight + 10)) {
+                            scope.overflow = [];
                         }
-                    };
-
+                    });
+
+                    // schemaTable is per table (needs to be done for record app with related tables)
+                    // rowIndex is per row in each table
+                    $timeout(function () {
+                        var uniqueIndex = scope.rowIndex;
+                        if (scope.config.containerIndex) {
+                            uniqueIndex = scope.config.containerIndex + "-" + uniqueIndex;
+                        }
+                        $rootScope.$on('aggLoaded-' + uniqueIndex, function(events, data) {
+                            var columnModelIndex = data;
+                            // +1 to account for the actions column
+                            var columnUiIndex = columnModelIndex + 1;
+                            // var hasPostLoadClass = scope.rowValues[columnModelIndex] && scope.rowValues[columnModelIndex].value.indexOf('chaise-post-load') > -1;
+                            var hasPostLoadClass =scope.rowValues[columnModelIndex].value.indexOf('chaise-post-load') > -1;
```

Comment:
Just to do extra check you should add `scope.rowValues[columnModelIndex].isHTML` to this.